pkg/tuf/delegations.go

```diff
@@ -0,0 +1,29 @@
+package tuf
+
+import (
+	"fmt"
+
+	"github.com/theupdateframework/notary/client"
+	"github.com/theupdateframework/notary/tuf/data"
+)
+
+// Delegate all paths ("*") to targets/releases.
+// https://github.com/theupdateframework/notary/blob/f255ae779066dc28ae4aee196061e58bb38a2b49/cmd/notary/delegations.go
+func delegateToReleases(repo client.Repository, publicKey data.PublicKey) error {
+	// the public keys used to verify the delegatee
+	publicKeys := []data.PublicKey{publicKey}
+	// the target paths entrusted to the delegatee
+	paths := make([]string, 2)
```

Comment:
This creates an array with 2 indexes allocated, and then you append two more, resulting in an array of length 4. I assume you meant this instead 
```suggestion
	paths := make([]string, 0, 2)
```